### PR TITLE
header renamed

### PR DESCRIPTION
**Pull request by harishkapa**

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -1,7 +1,7 @@
 <head></head>
 <body>
     <header>
-        <h1>Main Header</h1>
+        <h1>Header added</h1>
```
Comment (learning-machine1010) [inline index.html]: Rename this to Home Page

**Review by learning-machine1010 — CHANGES_REQUESTED** [review]
Make changes as requested

File: index.html (hunk 1 above)
Comment (harishkapa) [reply index.html]: changes done

**Review by learning-machine1010 — APPROVED** [review]
LGTM

**Pull request merged**